packages/server/src/notification/__tests__/notification.service.spec.ts

```diff
@@ -0,0 +1,105 @@
+import { HttpService } from '@nestjs/axios';
+import { ConfigService } from '@nestjs/config';
+import * as sinon from 'sinon';
+import { NotificationService } from '../notification.service';
+import { ProjectService } from '../../project/project.service';
+
+const sandbox = sinon.createSandbox();
+
+// Fixtures
+const MOCK_PROJECT = {
+  id: '1'
+};
+
+describe('NotificationService', () => {
+  let notificationService: NotificationService;
+  let mockConfigService: sinon.SinonStubbedInstance<ConfigService>;
+  let mockHttpService: sinon.SinonStubbedInstance<HttpService>;
+  let mockProjectService: sinon.SinonStubbedInstance<ProjectService>;
+
+  beforeEach(async () => {
+    mockConfigService = sandbox.createStubInstance(ConfigService);
+    mockConfigService.getOrThrow.returns('');
+    mockHttpService = sandbox.createStubInstance(HttpService);
+    mockProjectService = sandbox.createStubInstance(ProjectService);
+    notificationService = new NotificationService(mockConfigService, mockHttpService, mockProjectService);
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  it('should be defined', () => {
+    expect(notificationService).toBeDefined();
+  });
+
+  describe('Send Email', () => {
+    it('should send password reset email', async () => {
```

Comment:
This should be something like this:

```suggestion
  describe('Send Password Reset Email', () => {
    it('sends email successfully', async () => {
```